shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -1145,11 +1145,6 @@ - (BOOL)resignFirstResponder {
 }
 
 - (BOOL)canPerformAction:(SEL)action withSender:(id)sender {
-  // When scribble is available, the FlutterTextInputView will display the native toolbar unless
-  // these text editing actions are disabled.
-  if ([self isScribbleAvailable] && sender == NULL) {
-    return NO;
```

Comment:
@fbcouch Any idea if this is still doing anything important?

Reply:
It might be worth checking on iPadOS 16 vs 17 just to be sure (I have an iPad that's only on 16, so I can test that one once this is built, if needed), but it may be that other things in the engine or OS have changed to prevent the native toolbar from showing up

Reply:
Thanks, I think I have an iPad with 17, I'll try it out tonight and post back.

Reply:
I think i looked into this but forgot - in what other cases will `sender` be nil too? during scribble it seems? 

i vaguely remember when you enter memoji `sender` is nil, but could be wrong.

Reply:
I'm not sure, but yeah it looks like during Scribble. Flutter doesn't support selecting a memoji out of the box does it?

I forgot to link my response here (https://github.com/flutter/engine/pull/50923#issuecomment-1985073847), but I did try this out on a physical iOS 17 iPad and it seemed to work fine. I tried all of the Scribble features I could think of, plus normal soft keyboard text editing, including the text selection toolbar.

Reply:
> i vaguely remember when you enter memoji sender is nil, but could be wrong.

I think i remember it wrongly. memoji has nothing to do with this line. 

> Flutter doesn't support selecting a memoji out of the box does it?

Yes, but memoji is part of the keyboard that we don't control. So what we did was to make it a no-op when user choose their memoji. This is done with the `pasteboard.hasStrings` check below.